lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -91,6 +91,9 @@ class SwiftASTManipulatorBase {
     swift::VarDecl::Introducer GetVarIntroducer() const;
     bool GetIsCaptureList() const;
     bool IsMetadataPointer() const { return m_name.str().startswith("$τ"); }
+    bool IsOutermostMetadataPointer() const {
```

Comment:
Isn't depth 0 the innermost one?

Reply:
Just double-checked, 0 is the outermost depth. From the [Swift generics book](https://download.swift.org/docs/assets/generics.pdf): "The depth identifies a specific generic declaration, starting from zero for the top- level generic declaration and incrementing for each nested generic declaration."

Reply:
Good to know!